sdk/core/core-amqp/src/auth/sas.ts

```diff
@@ -29,69 +21,38 @@ export class SasTokenProvider implements TokenProvider {
    * @property {string} key - The secret value associated with the above EventHub/ServiceBus key.
    */
   key: string;
-  /**
-   * @property {number} tokenRenewalMarginInSeconds - The number of seconds within which it
-   * is good to renew the token. Default = 900 seconds (15 minutes).
-   */
-  tokenRenewalMarginInSeconds: number;
-  /**
-   * @property {number} tokenValidTimeInSeconds - The number of seconds for which the token
-   * is valid. Default = 3600 seconds (1 hour).
-   */
-  tokenValidTimeInSeconds: number;
 
   /**
-   * Initializes a new instance of SasTokenProvider
+   * Initializes a new instance of SharedKeyCredential
    * @constructor
-   * @param {string} namespace - The namespace of the EventHub/ServiceBus instance.
    * @param {string} keyName - The name of the EventHub/ServiceBus key.
    * @param {string} key - The secret value associated with the above EventHub/ServiceBus key
    */
-  constructor(
-    namespace: string,
-    keyName: string,
-    key: string,
-    tokenValidTimeInSeconds?: number,
-    tokenRenewalMarginInSeconds?: number
-  ) {
-    this.namespace = namespace;
+  constructor(namespace: string, keyName: string, key: string) {
```

Comment:
The constructor doesn't need namespace anymore...

Reply:
Ahhh!!

Reply:
All places that from where `SharedKeyCredential` is created should be updated as well